src/main/kotlin/com/supergrecko/questionbot/services/ConfigService.kt

```diff
@@ -2,11 +2,20 @@ package com.supergrecko.questionbot.services
 
 import com.supergrecko.questionbot.dataclasses.BotConfig
 import com.supergrecko.questionbot.dataclasses.GuildConfig
+import com.supergrecko.questionbot.services.getGuild as getConfig
 import me.aberrantfox.kjdautils.api.annotation.Service
 import me.aberrantfox.kjdautils.discord.Discord
 import me.aberrantfox.kjdautils.internal.di.PersistenceService
+import net.dv8tion.jda.api.entities.Guild
 import net.dv8tion.jda.internal.entities.TextChannelImpl
 
+data class QGuild(
```

Comment:
Instead of collecting the guild and the guild config separately via multiple service helpers we are able to grab both from a single helper.